app/views/dashboard/_navigation.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= render TabsComponent.new do |component| %>
+  <%= component.tab(path: dashboard_path) do %>
+    <strong>Overview</strong>
+  <% end %>
+  <%= component.tab(path: admin_supplier_users_path) do %>
```

Comment:
Uhh, does that path look wrong to you too?

Reply:
It does but then it doesn't really matter. I need to rebase once the report schools pr has been merged and that will be corrected then. Or perhaps I can copy your routes into mine and worry about the conflicts later